src/components/Property/Property.js

```diff
@@ -61,7 +61,7 @@ export default class Property extends PureComponent {
           <PropertyConstraints constraints={constraints} type={type} isRequired={isRequired} />
           {enumValues && this.renderEnumValues(enumValues)}
           {defaultValue !== undefined && this.renderDefaultValue(defaultValue)}
-          {description && <Description description={description} />}
+          {description && <div><Description isInline description={description} /></div>}
```

Comment:
Without the `div`, the description on property will be rendered inline, i.e. on the same line.

If I remove `isInLine`, then the result will be wrapped in a `p` tag, which breaks the tree css. The current tree css is not desirable, but I'd like to deal with that at a slightly later stage.